File: 프로그래머스/unrated/181901. 배열 만들기 1/배열 만들기 1.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n, int k) {
    vector<int> answer;
    int i = 1;
    while(i <= n)
    {
        if (i % k == 0) answer.push_back(i);
        i++;
    }
    return answer;
}

File: 프로그래머스/0/181901. 배열 만들기 1/배열 만들기 1.java
class Solution {
    public int[] solution(int n, int k) {
        int[] answer = new int[n / k];
        
        for (int i = k, j = 0; i <= n; i += k) {
            answer[j++] = i;
        }
        
        return answer;
    }
}